broker/src/main/java/io/zeebe/broker/Broker.java

```diff
@@ -253,9 +258,18 @@ private AutoCloseable subscriptionAPIStep(final BrokerInfo localBroker) {
         new SubscriptionApiCommandMessageHandlerService(localBroker, atomix);
     partitionListeners.add(messageHandlerService);
     scheduleActor(messageHandlerService);
+    addDiskSpaceUsageListener(messageHandlerService);
     return messageHandlerService;
   }
 
+  private void addDiskSpaceUsageListener(final DiskSpaceUsageListener diskSpaceUsageListener) {
+    if (diskSpaceUsageMonitor == null) {
```

Comment:
IF you would add as a separate last step the disk space monitor installation then you dont need this if.